doc/user/content/sql/functions/now_and_mz_now.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "now and mz_now functions"
```

Comment:
This file has content changes beyond search-and-replace. It's way better looking now without all the extract and _ms stuff!